hologram_dashboard/fetcher/BUILD

```diff
@@ -0,0 +1,19 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_binary(
+    name = "fetcher_main",
+    srcs = ["fetcher_main.cc"],
+    deps = [
+        "//fetcher:fetcher_lib"
+    ],
+)
+
+cc_library(
+    name = "fetcher_lib",
```

Comment:
-> hologram_data_source_availabiliity_fetcher

Making the lib name consistent with .cc file name.